frontend/apps/reader/modules/readerhandmade.lua

```diff
@@ -119,186 +120,189 @@ function ReaderHandMade:onToggleHandmadeFlows()
 end
 
 function ReaderHandMade:addToMainMenu(menu_items)
-    menu_items.handmade_toc = {
-        text = _("Custom table of contents") .. " " .. self.custom_toc_symbol,
-        checked_func = function() return self.toc_enabled end,
-        callback = function()
-            self:onToggleHandmadeToc()
-        end,
-    }
-    menu_items.handmade_hidden_flows = {
-        text = _("Custom hidden flows"),
-        checked_func = function() return self.flows_enabled end,
-        callback = function()
-            self:onToggleHandmadeFlows()
-        end,
-    }
-    --[[ Not yet implemented
-    menu_items.handmade_page_numbers = {
-        text = _("Custom page numbers"),
-        checked_func = function() return false end,
-        callback = function()
-        end,
-    }
-    ]]--
-    menu_items.handmade_settings = {
-        text = _("Custom layout features"),
-        sub_item_table_func = function()
-            return {
-                {
-                    text = _("About custom table of contents") .. " " .. self.custom_toc_symbol,
-                    callback = function()
-                        UIManager:show(InfoMessage:new{
-                            text = _([[
-If the book has no table of contents or you would like to substitute it with your own, you can create a custom TOC. The original TOC (if available) will not be altered.
+    if Device:isTouchDevice() then
```

Comment:
For an early return the code can be simplified like this:
```suggestion
    if Device:isTouchDevice() then
        return
    end
```

We (or at least I) prefer that legibility-wise. :-)

Reply:
``` lua
if Device:isTouchDevice() then
        EXISTING CODE
        return
end
```

that what you mean?

Reply:
Nope. ^_^ I mean exactly what I wrote. None of the changes, only an early return.

Reply:
Here's a random result that seems to offer a reasonable explanation as to why I often prefer it (if applicable): https://medium.com/swlh/return-early-pattern-3d18a41bba8

Reply:
The fact that it also solves the parallel diff/blame discussion is merely incidental to me but it happens to solve that as well. ;-)

Reply:
I am confused (sorry, might be due to inexperience), but that would not remove the options at all. can you be more specific?

Reply:
Return means it returns a value to the caller. Nothing can happen after that.

Reply:
```lua
if Device:isTouchDevice() then
        return
end
EXISTING CODE
```
but then the whole thing would run regardless, again pardon my inexperience.

Reply:
Nothing under return will run. But I see where I caused confusion, I forgot to update the condition as appropriate.
```suggestion
    if not Device:isTouchDevice() then
        return
    end
```

Reply:
I see, incidentally though you were suggesting to remove them from touch enabled devices 🤓. would have caused quite the stir.